candi/bashible/bashible.sh.tpl

```diff
@@ -104,13 +103,12 @@ function get_secret() {
 function get_bundle() {
   resource="$1"
   name="$2"
-  max_retries="$3"
 
   if type kubectl >/dev/null 2>&1 && test -f /etc/kubernetes/kubelet.conf ; then
     attempt=0
     until kubectl_exec get "$resource" "$name" -o json; do
       attempt=$(( attempt + 1 ))
-      if [ -n "${max_retries-}" ] && [ "$attempt" -gt "${max_retries}" ]; then
+      if [ -n "${MAX_RETRIES-}" ] && [ "$attempt" -gt "${MAX_RETRIES}" ]; then
```

Comment:
In fact, there isn't MAX_RETRIES at all?

Reply:
yes, that variable sets from command line